taxi/views.py

```diff
@@ -1,3 +1,20 @@
+from django.http import HttpResponse, HttpRequest
 from django.shortcuts import render
 
-# Create your views here.
+from taxi.models import Driver, Manufacturer, Car
+
+
+def index(request: HttpRequest) -> HttpResponse:
+    num_drivers = Driver.objects.count()
```

Comment:
Please assign those expressions as values directly to the context instead of creating variables